x-pack/plugins/monitoring/public/components/table/eui_table.js

```diff
@@ -37,6 +46,89 @@ export class EuiMonitoringTable extends React.PureComponent {
       return column;
     });
 
+    if (setupMode && setupMode.enabled) {
+      columns.push({
```

Comment:
I think this column might need custom sorting. Something like:
`sortable: (row) => get(setupMode, 'data.byUuid', {})[row.uuidField]`

I understand this is an object, so you might need to create a map to `isInternalCollector`, `isPartiallyMigrated`, and `isFullyMigrated `

Unless `uuidField` already correlates to `status` some how

Reply:
Yup, good point!

Reply:
Blocked by this: https://github.com/elastic/eui/issues/2012

Reply:
I can't mark these fields as sortable without having a concrete field to sort on. Currently, I'm reusing the `uuidField` for both, but this won't scale because both columns will be from the same field, yielding inconsistent results. I need to be able to sort based on a computed column (one without a `field`) but this appears to be a bug in EUI currently